src/Swift/Registry.php

```diff
@@ -25,18 +25,12 @@ class Registry
     /**
      * @var array
      */
-    protected $registry;
+    private $registry;
 
-    /**
-     * @param RegistryLoader $registryLoader
-     */
-    public function __construct($registryLoader = null)
+    public function __construct(string $registryFile = null)
     {
-        if (null === $registryLoader) {
-            $registryLoader = new RegistryLoader(__DIR__ . '/iban_registry.yaml');
-        }
-
-        $this->registry = $registryLoader->load();
+            $this->registry = include $registryFile
```

Comment:
`require` will throw a fatal error if the file cannot be found, and thus it feels like a better fit here. Because I can't imagine we could do anything useful for the case where the file is somehow missing.

```suggestion
            $this->registry = require $registryFile
```